doc/source/whatsnew/v1.5.0.rst

```diff
@@ -573,6 +573,10 @@ Performance improvements
 - Performance improvement when setting values in a pyarrow backed string array (:issue:`46400`)
 - Performance improvement in :func:`factorize` (:issue:`46109`)
 - Performance improvement in :class:`DataFrame` and :class:`Series` constructors for extension dtype scalars (:issue:`45854`)
+- Performance improvement in :class:`BusinessHour` ``str`` and ``repr``. (related to :issue:`44764`)
+- Performance improvement in datetime arrays string formatting when one of the default strftime formats ``"%Y-%m-%d %H:%M:%S"`` or ``"%Y-%m-%d %H:%M:%S.%f"`` is used. (:issue:`44764`)
+- Performance improvement in :meth:`Series.to_sql` and :meth:`DataFrame.to_sql` (:class:`SQLiteTable`) when processing time arrays. (related to :issue:`44764`)
```

Comment:
same. is this a significant change in to_sql? this seems just a side effect and not worth mentioning.

Reply:
65% improvement for writing `time` objects, I guess that it's significant enough to write a line here. 
https://github.com/pandas-dev/pandas/pull/46759#issuecomment-1097335128